design/vela-core/migrate-from-oam-runtime.md

```diff
@@ -0,0 +1,145 @@
+# Migrate from OAM Runtime
+
+Now we are refactoring OAM runtime in vela-core with CUE based abstractions, all source code from [oam-kubernetes-runtime](https://github.com/crossplane/oam-kubernetes-runtime)
+has already merged into [vela-core](https://github.com/oam-dev/kubevela/pull/663) now. Here is the doc for users who want
+to migrate from OAM Runtime to use KubeVela.
+
+## For Users Who are Using OAM runtime as Standalone Controller
+
+If you are using OAM Runtime as Standalone Controller, migrating to KubeVela is very straight forward.
+
+Server-side KubeVela(We call it `vela-core` for convenience) now include following **BUILT-IN** CRD and controllers.
+
+| Type |  CRD   | Controller  | From |
+| ---- |  ----  | ----  | ----  |
+| Framework | `applicationconfigurations.core.oam.dev` | Yes | OAM Runtime |
```

Comment:
Why do we call it `Framework`?

Reply:
Do you have any suggestions?

Reply:
Nope, I'm just curious about the meaning.

Reply:
Control Plane Objects